scripts/app.js

```diff
@@ -1,3 +1,82 @@
-// empty j file for awesome books scripts
-const hello = 1;
-hello('Nelly and Riley');
\ No newline at end of file
+import { getBooksFromStorage, saveBooksToStorage } from './storage.js';
+
+const dataSaved = getBooksFromStorage();
+const bookContainer = document.querySelector('.books-container');
+const form = document.querySelector('form');
+const titleInput = document.getElementById('title');
+const authorInput = document.getElementById('author');
+let booksCollection = [];
+
+if (dataSaved) {
+  booksCollection = dataSaved;
+} else {
+  booksCollection = [];
+}
+
+function addBookElementsToContainer() {
+  bookContainer.innerHTML = '';
+
+  const bookElements = booksCollection.map((book) => {
+    const bookElement = document.createElement('div');
+    bookElement.classList.add('book');
+
+    const titleElement = document.createElement('p');
+    titleElement.classList.add('book-title');
+    titleElement.textContent = book.title;
+
+    const authorElement = document.createElement('p');
+    authorElement.classList.add('book-author');
+    authorElement.textContent = book.author;
+
+    const removeButton = document.createElement('button');
+    removeButton.classList.add('btn', 'remove');
+    removeButton.textContent = 'Remove';
+
+    removeButton.addEventListener('click', () => {
```

Comment:
- [ ] Good job you did here 👏 . However, based on project requirements you're meant to create seperate functions for the `add` and `remove` function and then a sperate display collection to render the books. So ideally, you should have  `3` standalone functions to acomplish these task.
- [ ] This requirement is neccessery for modurality of your code, and needed for future task which would be asked of you 👍 

## Sample add function:
```
const addBook = (title, author) => {
  id = generateID(); // Optional: You can use title or author as  primary key instead of an ID in as much you check for uniqueness
  books = books.concat({ title, author, author });
};
```
## Sample remove function:
```
```javascript
const removeBook = (bookID) => {
  // bookID can be any other property, e.g. title or author, in as much it is unique
  books = books.filter(({ id }) => bookID !== id); // This assumes that `bookID` parameter is the book id
};
```

- Then use another mapping function or implementation of choice to display books 👍

Reply:
Thank you for this feedback.
We will make the necessary changes.
Regards.